README.md

```diff
@@ -1,22 +1,22 @@
 # Secure Loader Project
 
-The secure bootloader allows the end-user of MAX78000 to load an encrypted firmware to MAX78000 based platforms and lock the SWD port. Once it is programmed, the firmware, including the CNN model and application code, cannot be extracted from the chip by anyone except the developer of the firmware who has the keys.
+The secure bootloader allows the end user of MAX78000 to load an encrypted firmware to MAX78000-based platforms and lock the SWD port. Once it is programmed, the firmware, including the CNN model and application code, cannot be extracted from the chip by anyone, except the developer of the firmware who has the keys.
 
 ## Programming the [MAX78000](https://www.analog.com/en/products/max78000.html) with the Bootloader Firmware
 
 _Note: Install the [Analog Devices MSDK](https://www.analog.com/en/design-center/evaluation-hardware-and-software/software/software-download?swpart=SFW0010820A) before following the steps below!_
```

Comment:
I think we should link to these instructions [here](https://analogdevicesinc.github.io/msdk//USERGUIDE/#installation) instead